readme.md

```diff
@@ -16,11 +16,6 @@ We use a few tools to help with development. To install or update the required d
 
 `rake dependencies`
 
-#### CocoaPods
-
-Simplenote for iOS uses [CocoaPods](http://cocoapods.org/) to manage third party libraries.
-Third party libraries and resources managed by CocoaPods will be installed by the `rake dependencies` command above.
-
```

Comment:
Might be worth replacing this section with `#### SwiftPM` to indicate this project uses SwiftPM nowadays?

Reply:
For other reader's reference, in Simplenote iOS we agree to keep this off and to not mention SwiftPM. More in https://github.com/Automattic/simplenote-ios/pull/1716#discussion_r2149447959